app/models/ring.rb

```diff
@@ -0,0 +1,19 @@
+class Ring
+  def initialize(redirection_to_link)
+    @oldest_redirection = Redirection.order(created_at: :desc).last
```

Comment:
I think the goal was to clean this up a little later. This was a really fast fix for some circular dependency problems we were having. Wouldn't be surprised if this class gets totally refactored. Personally, I'd like to see `Ring.link(...)` that calls `new(...).link` as a convenience method.